src/main/java/core/basesyntax/strategy/impl/BirthdayDiscountService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class BirthdayDiscountService implements DiscountService {
+    private double discount = 33;

```

Comment:
no need to create this variable - you can return the value directly

**same for all other places**